### ISSUE TITLE
The term 'Import-PodeSnapIn' is not recognized when running pode as a windows service with nssm.

### ISSUE THREAD

**Issue by KuenzelIT** [Badgerati/Pode#594]
### Describe the Bug
When I try to run my pode webservice as a windows service with nssm, I get the following error:

```powershell
Import-PodeSnapIn -Name 'Microsoft.Exchange.Management.PowerShell .
~~~~~~~~~~~~~~~~~
The term 'Import-PodeSnapIn' is not recognized as the name of a cmdlet, function, script file, or
operable program. Check the spelling of the name, or if a path was included, verify that the path is
correct and try again.
```

It works well using `pode start`.

### Steps To Reproduce
1. Create the server.ps1:
```
Start-PodeServer -Threads 1 {
    $config = Get-PodeConfig

    Import-PodeSnapIn -Name 'Microsoft.Exchange.Management.PowerShell.SnapIn'

    Add-PodeEndpoint -Address "servername.com" -Port 8666 -Protocol Https -Certificate $config.certificateName
}
``` 

2. Install the server as a service (installService.ps1):
```powershell
$service = Import-PowerShellDataFile ($PSScriptRoot + "\config\service.psd1")

$exe = (Get-Command powershell.exe).Source
$name = $service.name
$file = $PSScriptRoot + '\server.ps1'
$logFile = $PSScriptRoot + '\logs\server.log'
$arg = "-ExecutionPolicy Bypass -NoProfile -Command `"$($file)`""

nssm install $name $exe $arg
nssm set $name AppStdout $logFile
nssm start $name
 ```

3. See the error in the logfile `logs/server.log`:
```powershell
Import-PodeSnapIn -Name 'Microsoft.Exchange.Management.PowerShell .
~~~~~~~~~~~~~~~~~
The term 'Import-PodeSnapIn' is not recognized as the name of a cmdlet, function, script file, or
operable program. Check the spelling of the name, or if a path was included, verify that the path is
correct and try again.
```

### Expected Behavior
I would expect the webservice to run without any error, since it works well using `pode start` in a fresh powershell (without adding any snap-in before).

### Platform
 - OS: Microsoft Windows Server 2016 Standard
 - Browser: none
 - Versions:
   - Pode: 1.8.1
   - PowerShell: 5.1.14393.3471

**Comment by Badgerati** [comment]
Hi @KuenzelIT,

I think I can see why this is happening, `Import-PodeSnapIn` appears to not be in the FunctionsToExport list.

I can get this fixed; for now you should be able to use `Import-Module -Name Pode` at the top of your `server.ps1`, and that should expose the function.

**Comment by KuenzelIT** [comment]
Thanks for the quick response!

I tested it using `Import-Module -Name Pode` but that does not seem to help. I'm still getting the same error message.

**Comment by Badgerati** [comment]
Gah, sometimes it works as a quick fix! 😂 

I managed to work it out anyway, and it is because it was missing in the module's FunctionsToExport. The reason `pode start` works is because the file at that point is invoked within Pode, so it would have seen all functions. The moment you try to invoke the script externally, it relies on the exported functions; causing the error.

I'll get this out as a 1.8.2.

**Comment by KuenzelIT** [comment]
Thank you for the super fast support und fix!

**Comment by Badgerati** [comment]
You're welcome! 😄 

This has been released in v1.8.2 now
